tests/test_logger_callback.py

```diff
@@ -0,0 +1,25 @@
+"""Test sampling from Bernoulli model."""
+import aiohttp
+import pytest
+
+import helpers
+
+program_code = "parameters {real y;} model {y ~ normal(0,1);}"
```

Comment:
Should we import these simple models from somewhere (e.g `helpers.py`)

Also do we want to use cached versions or always compile new models?

Reply:
The cached models get used automatically when they exist. This seems to work well. We know that it compiled successfully (at least once).

Reply:
Putting the program_code strings for common models in helpers does sound like a good idea. Maybe in a separate PR though.